### PR TITLE
[REVIEW] remove adjacency list conversions

### PR DESCRIPTION
**Pull request by Iroy30**

### CODE REVIEW

File: cpp/src/traversal/sssp.cu [hunk 1]
```diff
@@ -314,17 +313,6 @@ gdf_error gdf_sssp(gdf_graph* gdf_G,
                 GDF_INVALID_API_CALL);
   }
 
-  gdf_error err = gdf_add_adj_list(gdf_G);
-  if (err != GDF_SUCCESS)
-    return err;
-
-  GDF_REQUIRE(gdf_G->adjList->offsets->dtype == GDF_INT32,
```
[the remark this answers is not in the document]
Comment (Iroy30) [reply cpp/src/traversal/sssp.cu]: Added the dtype checks, source vert is checked in python

**Comment by BradReesWork** [comment]
Conflicts need to be resolved

**Comment by Iroy30** [comment]
rerun tests

**Comment by Iroy30** [comment]
rerun tests

**Comment by Iroy30** [comment]
rerun tests

**Comment by Iroy30** [comment]
rerun tests

**Pull request merged**